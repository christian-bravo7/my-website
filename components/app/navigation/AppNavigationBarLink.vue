<template>
  <NuxtLink
    class="mx-6 | text-center text-base | primary-text"
    exact-active-class="font-bold"
    v-bind="$attrs"
  >
    <slot />
  </NuxtLink>
</template>

<script lang="ts">
import { Vue, Component } from 'nuxt-property-decorator';

@Component
export default class AppNavigationBarLink extends Vue { }
</script>
