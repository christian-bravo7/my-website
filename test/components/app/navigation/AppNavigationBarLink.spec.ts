import { mount } from '@vue/test-utils';

import AppNavigationBarLink from '@/components/app/navigation/AppNavigationBarLink.vue';

describe('AppNavigationBarLink component', () => {
  it('should render correctly', () => {
    const wrapper = mount(AppNavigationBarLink, {
      propsData: {
        to: '/hello',
      },
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
