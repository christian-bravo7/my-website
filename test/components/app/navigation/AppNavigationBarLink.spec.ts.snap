// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppNavigationBarLink component should render correctly 1`] = `<a class="mx-6 | text-center text-base | primary-text" to="/hello"></a>`;
